atlasdb-dagger/src/main/java/com/palantir/atlasdb/services/KeyValueServiceModule.java

```diff
@@ -47,7 +49,9 @@ public KeyValueService provideWrappedKeyValueService(@Named("rawKvs") KeyValueSe
                                                          TimestampService tss,
                                                          ServicesConfig config) {
         KeyValueService kvs = NamespacedKeyValueServices.wrapWithStaticNamespaceMappingKvs(rawKvs);
-        kvs = new SweepStatsKeyValueService(kvs, tss);
```

Comment:
less code dupe would be nice, but this isn't your fault and it's probably out of scope for this PR